app/src/main/java/org/mozilla/fenix/shortcut/CreateShortcutFragment.kt

```diff
@@ -57,8 +57,13 @@ class CreateShortcutFragment : DialogFragment() {
     }
 
     private fun updateAddButtonEnabledState() {
-        add_button.isEnabled = shortcut_text.text.isNotEmpty()
-        add_button.alpha = if (shortcut_text.text.isNotEmpty()) ENABLED_ALPHA else DISABLED_ALPHA
+        add_button.isEnabled = isTextValid()
```

Comment:
If a string is empty (no characters), it will definitely be blank (whitespace characters only). You can simply replace `isNotEmpty` with `isNotBlank` without introducing the `isTextValid` method.

Reply:
seems valid 👍🏻